src/com/TTT/Board.java

```diff
@@ -91,4 +90,26 @@ private int maximumColumnValue(int columnNumber) {
         return ((sideLength() * sideLength()) - sideLength()) + columnNumber;
     }
 
+    public List<Integer> availableMoves() {
+        List<Integer> availableMoves = new ArrayList<>();
+
+        for (int i = 0; i < size(); i++) {
+            if (!isOccupied(i)) {
+                availableMoves.add(i);
+            }
+        }
+        return availableMoves;
+    }
+
+    @Override
+    public Object clone() {
+        Board clone = new Board(this.sideLength());
+        for (int i = 0; i < this.size(); i++){
+            if(isOccupied(i)){
```

Comment:
spacing here is in consistent with the rest of your code where you have put a space after the `if`.